datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -163,6 +164,7 @@ public class Coordinator implements ZkAdapter.ZkAdapterListener, MetricsAware {
 
   private static final long EVENT_THREAD_LONG_JOIN_TIMEOUT = 90000L;
   private static final long EVENT_THREAD_SHORT_JOIN_TIMEOUT = 3000L;
+  private static final long STOP_PROPAGATION_RETRY_MS = 5000L;
```

Comment:
Should we start the practice to add the reasoning for the new timer values?

Reply:
I can add comments for the newly introduced ones. Would that help?

Reply:
Done